transifex.yml

```diff
@@ -1,8 +1,122 @@
+    source_file: frontend-app-enterprise-public-catalog/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-app-enterprise-public-catalog/src/i18n/messages/<lang>.json'
+  
+  # frontend-app-gradebook
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-app-gradebook/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-app-gradebook/src/i18n/messages/<lang>.json'
+  
+  # frontend-app-learner-record
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-app-learner-record/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-app-learner-record/src/i18n/messages/<lang>.json'
+  
+  # frontend-app-learning
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-app-learning/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-app-learning/src/i18n/messages/<lang>.json'
+  
+  # frontend-app-library-authoring
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-app-library-authoring/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-app-library-authoring/src/i18n/messages/<lang>.json'
+  
+  # frontend-app-payment
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-app-payment/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-app-payment/src/i18n/messages/<lang>.json'
+    
+  # frontend-app-profile
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-app-profile/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-app-profile/src/i18n/messages/<lang>.json'
+    
+  # frontend-app-program-console
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-app-program-console/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-app-program-console/src/i18n/messages/<lang>.json'
+    
+  # frontend-app-support-tools
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-app-support-tools/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-app-support-tools/src/i18n/messages/<lang>.json'
+    
+  # frontend-component-footer
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-component-footer/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-component-footer/src/i18n/messages/<lang>.json'
+    
+  # frontend-component-header
+  - filter_type: file
+    file_format: KEYVALUEJSON
+    source_language: en
+    source_file: frontend-component-header/src/i18n/transifex_input.json
+    translation_files_expression: 'frontend-component-header/src/i18n/messages/<lang>.json'
```

Comment:
i was hoping there might be a way to do this without copypasta (something like `frontend-*/src/i18n/transifex_input.json`), but that's out of scope for this